core/eolearn/tests/test_core_tasks.py

```diff
@@ -148,28 +149,68 @@ def test_io_task_pickling(filesystem, task_class):
     assert isinstance(unpickled_task, task_class)
 
 
-def test_add_rename_remove_feature(patch):
+def test_add_feature(patch: EOPatch) -> None:
     cloud_mask = np.arange(10).reshape(5, 2, 1, 1)
-    feature_name = "CLOUD MASK"
-    new_feature_name = "CLM"
+    feature = (FeatureType.MASK, "CLOUD MASK")
 
-    patch = copy.deepcopy(patch)
+    with pytest.raises(KeyError):
+        patch[feature]
 
-    patch = AddFeatureTask((FeatureType.MASK, feature_name))(patch, cloud_mask)
-    assert np.array_equal(patch.mask[feature_name], cloud_mask), "Feature was not added"
+    patch = AddFeatureTask(feature)(patch, cloud_mask)
+    assert np.array_equal(patch[feature], cloud_mask)
 
-    patch = RenameFeatureTask((FeatureType.MASK, feature_name, new_feature_name))(patch)
-    assert np.array_equal(patch.mask[new_feature_name], cloud_mask), "Feature was not renamed"
-    assert feature_name not in patch[FeatureType.MASK], "Old feature still exists"
 
-    patch = RemoveFeatureTask((FeatureType.MASK, new_feature_name))(patch)
-    assert feature_name not in patch.mask, "Feature was not removed"
+@pytest.mark.parametrize(
+    "feature, feature_data",
+    [
+        ((FeatureType.BBOX, None), BBox((24.54, 56.45, 95.4, 13.43), CRS(3857))),
+        ((FeatureType.TIMESTAMP, None), [datetime(2022, 1, 1, 10, 4, 7), datetime(2022, 1, 4, 10, 14, 5)]),
+    ],
+)
+def test_add_bbox_timestamps(feature: FeatureSpec, feature_data: np.ndarray) -> None:
+    # this test should fail for bbox and timestamps after rework
+    patch = EOPatch()
+    assert not patch[feature]
```

Comment:
I didn't know that you could test feature existence for bbox and timestams like this. TBH it feels wrong and scary, since I would expect this to crash due to the features not being present. But I see that in case of bbox and timestamps it really doesn't crash because they are [handled differently](https://github.com/sentinel-hub/eo-learn/blob/fix/readthedocs-links-not-working/core/eolearn/core/eodata.py#L250-L251):

```
...
bbox: Optional[BBox] = attr.ib(default=None)
timestamp: List[dt.datetime] = attr.ib(factory=list)
...
```

I would still much rather prefer to see `assert not feature in patch` used here.

Reply:
done